x-pack/plugins/security_solution/public/detections/components/alerts_table/index.tsx

```diff
@@ -355,6 +355,7 @@ export const AlertsTableComponent: React.FC<AlertsTableComponentProps> = ({
   }, [dispatch, filterManager, tGridEnabled, timelineId]);
 
   const leadingControlColumns = useMemo(() => getDefaultControlColumn(ACTION_BUTTON_COUNT), []);
+  const memoizedCellActions = useMemo(() => defaultCellActions, []);
```

Comment:
Given that `defaultCellActions` is imported and not re-evaluated I think it might be safe to not use `useMemo` here, wdyt?